tmcprototype/SubarrayNode/test/SubarrayNode_test.py

```diff
@@ -99,73 +99,67 @@ def test_State(self, tango_context):
-    #     assert tango_context.device.obsState == 2
-    #     # with pytest.raises(tango.DevFailed):
-    #     #     tango_context.device.EndScan()
-    #     # assert CONST.ERR_DUPLICATE_END_SCAN_CMD in tango_context.device.activityMessage
-    #     # PROTECTED REGION END #    //  SubarrayNode.test_EndScan
-
-    # def test_EndSB(self, tango_context):
-    #     """Test for EndSB command."""
-    #     # PROTECTED REGION ID(SubarrayNode.test_EndSB) ENABLED START #
-    #     tango_context.device.EndSB()
-    #     time.sleep(2)
-    #     assert tango_context.device.ObsState == CONST.OBS_STATE_ENUM_IDLE
-    #     # PROTECTED REGION END #    //  SubarrayNode.test_EndSB
+    def test_healthState(self, tango_context):
+        """Test for healthState"""
+        # PROTECTED REGION ID(SubarrayNode.test_healthState) ENABLED START #
+        assert tango_context.device.healthState == 1
+        # PROTECTED REGION END #    //  SubarrayNode.test_healthState
+
+    def test_AssignResources(self, tango_context):
+        """Test for AssignResources"""
+        # PROTECTED REGION ID(SubarrayNode.test_AssignResources) ENABLED START #
+        receptor_list = ['0001']
+        tango_context.device.AssignResources(receptor_list)
+        time.sleep(10)
+        assert tango_context.device.State() == DevState.ON
+        assert len(tango_context.device.receptorIDList) == 1
+        assert tango_context.device.obsState == 0
+        # PROTECTED REGION END #    //  SubarrayNode.test_AssignResources
+
+    def test_Configure(self, tango_context, create_dish_proxy):
+        """Test for Configure"""
+        # PROTECTED REGION ID(SubarrayNode.test_Configure) ENABLED START #
+        tango_context.device.Configure('{"scanID":12345,"pointing":{"target":{"system":"ICRS","name":'
+                                       '"Polaris","RA":"02:31:49.0946","dec":"+89:15:50.7923"}},"dish":'
+                                       '{"receiverBand":"1"},"csp":{"frequencyBand":"1","fsp":[{"fspID":1,'
+                                       '"functionMode":"CORR","frequencySliceID":1,"integrationTime":1400,'
+                                       '"corrBandwidth":0}]},"sdp":{"configure":'
+                                       '[{"id":"realtime-20190627-0001","sbiId":"20190627-0001","workflow":'
+                                       '{"id":"vis_ingest","type":"realtime","version":"0.1.0"},"parameters":'
+                                       '{"numStations":4,"numChannels":372,"numPolarisations":4,'
+                                       '"freqStartHz":0.35e9,"freqEndHz":1.05e9,"fields":{"0":'
+                                       '{"system":"ICRS","name":"Polaris","ra":0.662432049839445,'
+                                       '"dec":1.5579526053855042}}},"scanParameters":{"12345":'
+                                       '{"fieldId":0,"intervalMs":1400}}}]}}')
+        time.sleep(65)
+        assert tango_context.device.obsState == 2
```

Comment:
Instead of using numbers can use constants defined in CONST file.

Reply:
DONE.